### PR TITLE
use kubectl create deployment to create deployment with  --replicas and --port

### PR DESCRIPTION
**Pull request by Cweiping**
[kubectl run ](https://github.com/kubernetes/kubernetes/issues/89651) become create pod ，so upgrade document of English

Fix Issue：https://github.com/kubernetes/kubernetes/issues/91064

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/namespaces.md [hunk 1]
```diff
@@ -189,7 +189,7 @@ This delete is asynchronous, so for a time you will see the namespace in the `Te
     To demonstrate this, let's spin up a simple Deployment and Pods in the `development` namespace.
 
     ```shell
-    kubectl create deployment snowflake --image=k8s.gcr.io/serve_hostname  -n=development
+    kubectl create deployment snowflake --image=k8s.gcr.io/serve_hostname  -n=development --replicas=2
     kubectl scale deployment snowflake --replicas=2 -n=development
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/namespaces.md:193]: Why this change?
I don't understand it. I'm imagining that the reader will wonder why they are scaling _snowflake` from 2 to 2.

Comment (tengqm) [reply content/en/docs/tasks/administer-cluster/namespaces.md:193]: good catch
/lgtm cancel

Comment (Cweiping) [reply content/en/docs/tasks/administer-cluster/namespaces.md]: Sorry, I forgot to submit new changes. In v1.19, `kubectl create deployment` can already specify replicas and env, hoping to improve the use experience.

File: content/en/docs/reference/kubectl/docker-cli-to-kubectl.md [hunk 2]
```diff
@@ -56,8 +62,7 @@ service "nginx-http" exposed
 
 By using kubectl, you can create a [Deployment](/docs/concepts/workloads/controllers/deployment/) to ensure that N pods are running nginx, where N is the number of replicas stated in the spec and defaults to 1. You can also create a [service](/docs/concepts/services-networking/service/) with a selector that matches the pod labels. For more information, see [Use a Service to Access an Application in a Cluster](/docs/tasks/access-application-cluster/service-access-application-cluster).
 
-By default images run in the background, similar to `docker run -d ...`. To run things in the foreground, use:
-
+By default images run in the background, similar to `docker run -d ...`. To run things in the foreground, use [`kubectl run`](/docs/reference/generated/kubectl/kubectl-commands/#run) to create pod:
```
Comment (sftim) [inline content/en/docs/reference/kubectl/docker-cli-to-kubectl.md:65]:
```suggestion
When you create a Deployment, the containers run in the background. This is roughly similar to `docker run -d ...` with extra features.
To wait for the container to exit, use [`kubectl run`](/docs/reference/generated/kubectl/kubectl-commands/#run) to create a Pod and attach to its container:
```

and also (_I think_) specify `--attach` for `kubectl run`.

File: content/en/docs/tasks/administer-cluster/namespaces.md [hunk 3]
```diff
@@ -189,8 +189,7 @@ This delete is asynchronous, so for a time you will see the namespace in the `Te
     To demonstrate this, let's spin up a simple Deployment and Pods in the `development` namespace.
 
     ```shell
-    kubectl create deployment snowflake --image=k8s.gcr.io/serve_hostname  -n=development
-    kubectl scale deployment snowflake --replicas=2 -n=development
+    kubectl create deployment snowflake --image=k8s.gcr.io/serve_hostname  -n=development --replicas=2
     ```
     We have just created a deployment whose replica size is 2 that is running the pod called `snowflake` with a basic container that just serves the hostname.
     Note that `kubectl run` creates deployments only on Kubernetes cluster >= v1.2. If you are running older versions, it creates replication controllers instead.
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/namespaces.md:195]: Also remove this line and the one after it. Both generators and ReplicationController resources are superseded.

File: content/en/docs/reference/kubectl/docker-cli-to-kubectl.md (hunk 2 above)
Comment (Cweiping) [reply content/en/docs/reference/kubectl/docker-cli-to-kubectl.md:65]: > and also (_I think_) specify `--attach` for `kubectl run`.

[It](https://github.com/kubernetes/website/pull/21707/commits/cf3becadff9eecc03e995a3f0ccfead0b2487eac#diff-16decda489c4f33805ead292d4af10ceR70) has been specially introduced.

File: content/en/docs/tasks/administer-cluster/namespaces.md (hunk 3 above)
Comment (Cweiping) [reply content/en/docs/tasks/administer-cluster/namespaces.md]: Done

File: content/en/docs/concepts/policy/resource-quotas.md [hunk 4]
```diff
@@ -494,8 +494,7 @@ kubectl create quota test --hard=count/deployments.extensions=2,count/replicaset
 ```
 
 ```shell
-kubectl create deployment nginx --image=nginx --namespace=myspace
-kubectl scale deployment nginx --replicas=2 --namespace=myspace
+kubectl create deployment nginx --image=nginx --namespace=myspace --replicas=2
```
Comment (armujahid) [inline content/en/docs/concepts/policy/resource-quotas.md:497]: I think replicas flag is not yet supported by `kubectl create deployment`.
